spec/db/models.rb

```diff
@@ -27,24 +27,45 @@ class Author < VitualTotalTestBase
   virtual_total :total_recently_published_books, :recently_published_books
   virtual_aggregate :sum_recently_published_books_rating, :recently_published_books, :sum, :rating
 
-  # virtual_total using a virtual_has_many
+  # virtual_total using a virtual_has_many with an array of associations
```

Comment:
Is this really a `virtual_total`?  Did I add this comment?  If so... sorry...

Reply:
Looks like I didn't add this:

https://github.com/ManageIQ/activerecord-virtual_attributes/commit/9308002c7dd9c3bd4e42c32c552f0e63feedf8bc#diff-4520e22fc7e8d7545816ca398c24a778R55

But also, it isn't a `virtual_total`...

https://github.com/ManageIQ/activerecord-virtual_attributes/commit/9308002c7dd9c3bd4e42c32c552f0e63feedf8bc#diff-4520e22fc7e8d7545816ca398c24a778R61

So yeah...

Reply:
I made the comment more clear. this is here to be referenced by a virtual total